qa/pull-tester/rpc-tests.py

```diff
@@ -39,8 +39,8 @@
 #If imported values are not defined then set to zero (or disabled)
 if 'ENABLE_WALLET' not in vars():
     ENABLE_WALLET=0
-if 'ENABLE_BITCOIND' not in vars():
-    ENABLE_BITCOIND=0
+if 'ENABLE_DASHD' not in vars():
```

Comment:
Should not dashify env vars (or should dashify them everywhere but in another PR).